mk/dotnet-packages-build.sh

```diff
@@ -130,13 +130,13 @@ echo "INFO:	Performing main build tasks..."
 
 run_msbuild()
 {
-  MSBuild.exe /nologo /m /verbosity:minimal /p:Configuration=Release /p:TargetFrameworkVersion=v4.6 /property:PlatformToolset=v120 $*
+  C:/"Program Files (x86)"/MSBuild/14.0/bin/MSBuild.exe /nologo /m /verbosity:minimal /p:Configuration=Release /p:TargetFrameworkVersion=v4.6 /property:PlatformToolset=v140 /p:FrameworkPathOverride="C:/Program Files (x86)/Reference Assemblies/Microsoft/Framework/.NETFramework/v4.6" $*
```

Comment:
Would it be possible to use an environment variable here, instead of C:/"Program Files (x86)"?